components/Landing.tsx

```diff
@@ -22,7 +22,7 @@ export default function Landing({ onStart }: LandingProps) {
   }, []);
 
   return (
-    <div className="relative flex flex-col justify-start items-center h-[100dvh] md:h-screen bg-custom">
+    <div className="relative flex flex-col justify-start items-center h-[100dvh] md:h-screen  bg-custom">
```

Comment:
Dont forget to run Prettier before pushing the code.

Reply:
Will focus on the Redux in the next review👍